rotkehlchen/data_import/importers/bitcoin_tax.py

```diff
@@ -0,0 +1,285 @@
+import csv
+import hashlib
+import logging
+from pathlib import Path
+from typing import Any, Literal, Optional
+
+from rotkehlchen.accounting.structures.balance import AssetBalance, Balance
+from rotkehlchen.accounting.structures.base import HistoryEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.converters import LOCATION_TO_ASSET_MAPPING, asset_from_common_identifier
+from rotkehlchen.constants import ZERO
+from rotkehlchen.data_import.utils import BaseExchangeImporter, UnsupportedCSVEntry
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.asset import UnknownAsset
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import (
+    deserialize_asset_amount,
+    deserialize_timestamp_from_date,
+)
+from rotkehlchen.types import Fee, Location, TimestampMS
+from rotkehlchen.utils.misc import ts_sec_to_ms
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+CSVType = Literal['trades', 'other_events']
```

Comment:
It's okay to do this, but also keep in mind that for a type in just 2 places it's probaly not important to avoid the duplication. Could just put it as is.